tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -635,6 +636,55 @@ func (s *storeImpl) Delete(ctx context.Context, {{template "paramList" $pks}}) e
 }
 {{- end}}
 
+
+{{- if not .JoinTable }}
+// DeleteByQuery removes the objects based on the passed query
+func (s *storeImpl) DeleteByQuery(ctx context.Context, query *v1.Query) error {
+    {{- if not $inMigration}}
+	defer metrics.SetPostgresOperationDurationTime(time.Now(), ops.Remove, "{{.TrimmedType}}")
+    {{- end}}{{/* if not .inMigration */}}
+
+    var sacQueryFilter *v1.Query
```

Comment:
Move this down to where it is actually used.

Reply:
You actually do not need to declare it here. You can use short-var declaration because the `if-else` clause ensures the var is always init.

Reply:
So the conjunction portion always has the var so it needs to be declared. Also, all of the other sac portions do it this way so want to keep it consistent